domain/src/main/java/org/buffer/android/boilerplate/domain/interactor/browse/GetBufferoos.kt

```diff
@@ -14,9 +14,9 @@ import javax.inject.Inject
 open class GetBufferoos @Inject constructor(val bufferooRepository: BufferooRepository,
                                             threadExecutor: ThreadExecutor,
                                             postExecutionThread: PostExecutionThread):
-        SingleUseCase<List<Bufferoo>, Void?>(threadExecutor, postExecutionThread) {
+        SingleUseCase<List<Bufferoo>, Nothing?>(threadExecutor, postExecutionThread) {
```

Comment:
Why not just use Unit? Although Nothing is similar, but its really not the use case for it